src/main/java/pageObjects/CustomerPortal/SupportPortalPage.java

```diff
@@ -0,0 +1,110 @@
+package pageObjects.CustomerPortal;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import pageObjects.Admin_Portal.AdminPortalTest2ndPage;
+import webdriverbase.AppPage;
+
+
+public class SupportPortalPage extends AppPage {
+
+    public SupportPortalPage(WebDriver driver) {
+        super(driver);
+    }
+
+
+    public void navigateToHappyFoxSupportPortalURL(String url) {
+        this.driver.get(url);
+    }
+
+    @FindBy(id = "id_subject")
+    private WebElement Subject;
+
+    public void enterSubject(String text) {
+        Subject.sendKeys(text);
+    }
+
+    @FindBy(xpath = "//div[@class='cke_wysiwyg_div cke_reset cke_enable_context_menu cke_editable cke_editable_themed cke_contents_ltr cke_show_borders']")
+    private WebElement Message;
+
+    public void enterMessage(String text) {
+        Message.sendKeys(text);
+    }
+
+    @FindBy(id = "add-cc")
+    private WebElement AddCC;
+
+    public void clickAddCC() {
+        AddCC.click();
+    }
+
+    @FindBy(id = "add-bcc")
+    private WebElement AddBCC;
+
+    public void clickAddBCC() {
+        AddBCC.click();
+    }
+
+    @FindBy(xpath = "//input[@id='id_cc']")
+    private WebElement CC;
+
+    public void enterCC(String text) {
+        CC.sendKeys(text);
+    }
+
+    @FindBy(xpath = "//input[@id='id_bcc']")
+    private WebElement BCC;
+
+    public void enterBCC(String text) {
+        BCC.sendKeys(text);
+    }
+
+
+    @FindBy(xpath = "//a[@class='hf-attach-file_link']")
+    private WebElement BrowseFile;
+
+    public void addingScreenshot(String abc) {
+        BrowseFile.sendKeys(getTestDataFullDirPath(abc));
+
+    }
+
+
+    @FindBy(id = "id_name")
+    private WebElement FullName;
+
+    public void enterFullName(String text) throws InterruptedException {
+        sleep();
```

Comment:
It's not a good practice to use Thread.sleep. Rather try using implicit & explicitWaits